src/presentation/http/router/note.ts

```diff
@@ -372,6 +372,61 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
     });
   });
 
+  /**
+   * Create note relation by id.
+   */
+  fastify.post<{
+    Params: {
+      notePublicId: NotePublicId;
+    };
+    Body: {
+      parentNoteId: NotePublicId;
+    };
+    Reply: {
+      isCreated: boolean;
+    };
+  }>('/:notePublicId/relation', {
+    schema: {
+      params: {
+        notePublicId: {
+          $ref: 'NoteSchema#/properties/id',
+        },
+      },
+      body: {
+        parentNoteId: {
+          $ref: 'NoteSchema#/properties/id',
+        },
+      },
+      response: {
+        '2xx': {
+          type: 'object',
+          description: 'Was the relation created',
+          properties: {
+            isCreated: {
+              type: 'boolean',
+            },
+          },
+        },
+      },
```

Comment:
i guess status code 2xx already shows that relation had beed added, so isCreated field is unused